config.json

```diff
@@ -2357,6 +2357,28 @@
           "macros"
         ],
         "difficulty": 8
+      },
+      {
+        "slug": "pov",
+        "name": "POV",
+        "uuid": "ABC93451-451D-4497-B590-68CBF82E0A20",
+        "prerequisites": [
+          "pattern-matching",
+          "case",
+          "if",
+          "errors",
+          "recursion",
+          "enum",
+          "protocols",
+          "nil",
+          "structs",
+          "multiple-clause-functions"
+        ],
+        "practices": [
+          "recursion",
+          "pattern-matching"
+        ],
+        "difficulty": 9
```

Comment:
This is exciting 🤩 looks like this will be our most difficult exercise.

Reply:
I have not done many of the difficulty 8, but it definitely is one step above Zipper, which is 8.
Forth is a 10 though!

Reply:
I think difficult 9 is okay, we don't have a firm definition what each number means anymore, so it's just to give a general feel of the complexity